pkg/app/rds_postgres.go

```diff
@@ -413,3 +442,40 @@ func makeYamlList(dbs []string) string {
 	}
 	return dbsYaml
 }
+
+func (pdb RDSPostgresDB) execCommand(ctx context.Context, command []string) (string, string, error) {
+	podName, containerName, err := kube.GetPodContainerFromDeployment(ctx, pdb.cli, pdb.namespace, pdb.testWorkloadName)
+	if err != nil || podName == "" {
+		return "", "", err
+	}
+	return kube.Exec(pdb.cli, pdb.namespace, podName, containerName, command, nil)
+}
+
+// testWorkload creates Deployment Resource Manifest from which postgres command can be executed
+func testWorkload(ctx context.Context, name string, image string, namespace string) *appsv1.Deployment {
```

Comment:
Is this utility going to be used by other RDS apps as well? If yes, should we put this in a common util file?

Reply:
I think we should also change the name here, instead of testworkload, maybe we should have `jumpWorkload` or `bastionWorkload`.